Proofread files in the /configuration/firewall directory (#1810)
vyos/vyos-documentation: docs/configuration/firewall/global-options.rst

Overview
********

> Some firewall settings are global and affect entire systems. This section

Reviewer: ```suggestion
Some firewall settings are global and affect the entire system. This section
```

The "system" in this context is VyOS (well, its firewall subsystem). Since those options will only ever affect one system (the VyOS system it's set on), I don't think the plural form is appropriate here.

   .. code-block:: none

      set firewall global-options all-ping enable

   When you set this command, VyOS answers every ICMP echo request addressed
   to itself, but that response occurs only if no other rule drops or rejects
   local echo requests. In case of conflict, VyOS does not answer ICMP echo
   requests.

   .. code-block:: none

      set firewall global-options all-ping disable

   When you set this command, VyOS answers no ICMP echo requests addressed to
   itself, regardless of where they come from or what specific rules accept
   them.

.. cfgcmd:: set firewall global-options apply-to-bridged-traffic [ipv4 | ipv6]

   Apply IPv4 or IPv6 firewall rules to bridged traffic.

.. cfgcmd:: set firewall global-options broadcast-ping [enable | disable]

   Enable or disable the response to ICMP broadcast messages. The system
   alters the following parameter:

   * ``net.ipv4.icmp_echo_ignore_broadcasts``

.. cfgcmd:: set firewall global-options ip-src-route [enable | disable]
.. cfgcmd:: set firewall global-options ipv6-src-route [enable | disable]

   Set whether VyOS accepts packets with a source route option. The system

>    alters the following parameters:

Reviewer: ```suggestion
   Set whether VyOS accepts packets with a source route option.
   The following sysctl parameters will be changed:
```

The original wording was correct. Setting the option will set certain system parameters (those sysctl options mentioned below).

.. cfgcmd:: set firewall global-options ipv6-receive-redirects
   [enable | disable]

   Enable or disable VyOS to accept ICMPv4 or ICMPv6 redirect messages.

>    The system alters the following parameters:

Reviewer: ```suggestion
   Allow VyOS to accept ICMPv4 and ICMPv6 redirect messages.
   The following sysctl parameters will be changed:
```

   * ``net.ipv4.conf.all.accept_redirects``
   * ``net.ipv6.conf.all.accept_redirects``

.. cfgcmd:: set firewall global-options send-redirects [enable | disable]

   Enable or disable VyOS to send ICMPv4 redirect messages. The system alters

>    the following parameter:

Reviewer: ```suggestion
   Allow VyOS to send ICMPv4 redirect messages.
   The following sysctl parameter will be changed:
```

   * ``net.ipv4.conf.all.send_redirects``

.. cfgcmd:: set firewall global-options log-martians [enable | disable]

   Enable or disable logging of martian IPv4 packets. The system alters the

>    following parameter:

Reviewer: Please change every instance of "The system alters" to the appropriate wording shown above.
